packages/paste-core/components/datagrid/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
+import {Button} from '@twilio-paste/button';
+import {fireEvent, render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+// @ts-ignore typescript doesn't like js imports
+import axe from '../../../../../.jest/axe-helper';
+
+import {ComposableCells} from '../stories/ComposableCells';
+
+describe('Datagrid', () => {
```

Comment:
I think generally this looks like it covers everything we need, just needs a little tidy up.

I can't think of any interaction cases outside of these right now.

How do you feel about some tests that assert the markup? Specifically which aria attributes are set on what cells, just to ensure our markup output stays as expected?